crates/json/Cargo.toml

```diff
@@ -5,17 +5,24 @@ authors = ["Estuary Technologies, Inc"]
 edition = "2018"
 
 [dependencies]
+addr = {version="0.15.4", features=["idna", "std","psl"]}
 # TODO(johnny): bitvec had a breaking 1.0 release we've not upgraded to yet.
 bitvec = "0.19"
+bytecount = "0.6.3"
 fancy-regex = "*"
 fxhash = "*"
+iri-string = "0.6.0"
 itertools = "*"
+json-pointer = "0.3.4"
+lazy_static = "1.4.0"
 percent-encoding = "*"
 serde = "*"
 serde_json = "*"
 thiserror = "*"
+time = {version = "0.3.13", features=["macros", "formatting", "parsing"]}
```

Comment:
Elsewhere in Flow we've used the `chrono` crate, and it'd be kinda nice to stick with one such library if possible. This is definitely not a big deal though. If it'd be easy enough to consolidate time crates, then it seems like a good idea, but it's by no means a blocker.

Reply:
Yea, which should we be doing? I get the impression that the community is coalescing around `time` but what do you think?

Reply:
I don't have an opinion on this. Happy to replace time with chrono if that's where we want to go